pytorch/hf-genai-requirements.txt

```diff
@@ -0,0 +1,21 @@
+SentencePiece==0.2.0
+accelerate==0.28.0
+datasets==2.19.0
+einops==0.7.0
+evaluate==0.4.1
+mkl-include==2023.2.0
+mkl==2023.2.0
+nltk==3.8.1
+onnxruntime-extensions==0.10.1
+onnxruntime==1.17.3
+peft==0.10.0
+protobuf==4.24.4
+psutil==5.9.5
+py-cpuinfo==9.0.0
+rouge_score==0.1.2
+tokenizers==0.19.1
+torch>=1.3
```

Comment:
This is problematic. If anything, I'd pin the version for `torch`.

Reply:
You should actually remove any component in here that is in another requirement.txt file, since we install torch in the base it doesn't need to be here.

Reply:
In other words, make sure there are no duplicates in `python/requirements.txt`, `pytorch/requirements.txt`, and `pytorch/multinode-requirements.txt`.